### PR TITLE
Web Lab Investigation: Use data_json field

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Quick fix to use the "data_json" field so the data gets ingested into Redshift

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/weblab/CdoBramble.js [hunk 1]
```diff
@@ -10,12 +10,15 @@ import firehoseClient from '../lib/util/firehose';
 
 // Some temporary logging to diagnose possible loading issues.
 function tempLog(event, data = null) {
+  const dataJson = {
```
Comment (breville) [inline apps/src/weblab/CdoBramble.js]: nit: this isn't json yet :)

File: apps/src/weblab/CdoBramble.js [hunk 2]
```diff
@@ -10,12 +10,15 @@ import firehoseClient from '../lib/util/firehose';
...
+    is_cros: /\bCrOS\b/.test(navigator.userAgent),
```
Comment (breville) [inline apps/src/weblab/CdoBramble.js]: I think we might actually get the user agent separately, so this was done as a convenience, but might not be necessary.

**Review by breville — APPROVED** [review]
Thanks so much for this.  My bad for not realizing the field name was so important.

File: apps/src/weblab/CdoBramble.js (hunk 1 above)
Comment (sanchitmalhotra126) [reply apps/src/weblab/CdoBramble.js]: Haha true, just wanted to call it something other than "data"

File: apps/src/weblab/CdoBramble.js (hunk 2 above)
Comment (sanchitmalhotra126) [reply apps/src/weblab/CdoBramble.js]: Oh yep you're right - this shows up in the "device" payload.

**Pull request merged**